Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/ProjectFileHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Amazon.Lambda.Annotations.SourceGenerator.FileIO;
+using System;
+using System.Xml;
+
+namespace Amazon.Lambda.Annotations.SourceGenerator
+{
+    /// <summary>
+    /// Utilities for working with project (.csproj) files, such as allowing projects
+    /// to opt out of the CloudFormation description modification.
+    /// </summary>
+    public class ProjectFileHandler
+    {
+        private const string OptOutNodeXpath = "//PropertyGroup/AWSSuppressCloudFormationDescriptionModification";
```

Comment:
nit: similar to my above comments, i'd prefer if we make the attribute name shorter and refer to what it's purpose actually is instead of an implementation detail. So maybe name it `AWSSuppressTelemetry`?

Reply:
Went with `AWSSuppressLambdaAnnotationsTelemetry` after internal discussion. Fixed in amended commit.